.tours/accessibility-improvements.tour

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Accessibility Improvements",
+  "steps": [
+    {
+      "title": "Introduction",
+      "description": "We evaluated the accessibility of the Visual Studio Code CodeTour extension for blind and low-vision (BLV) users. Based on our findings, we made prototype-quality changes to CodeTour to improve accessibility. Finally, we identified future work items that could make CodeTour and Visual Studio Code more accessible. This tour contains the changes we made. "
+    },
+    {
+      "file": "src/player/a11yhelpers.ts",
+      "description": "This function allow us to determine if user has accessibility support enabled in their settings. This setting controls if the VS Code should run in a mode where it is optimized for screen readers. All our changes are behind this flag to make sure we are respecting the user's settings. ",
+      "line": 3
+    },
+    {
+      "file": "src/player/a11yhelpers.ts",
+      "description": "Thi util is is used to show announcement as the BLV user is navigating the tour. Our sponsor suggested that we use this when we faced obstacles while trying to make the Comment class more accessible.  ",
+      "line": 11
+    },
+    {
+      "file": "package.json",
+      "description": "This is the \"Go To Tour Step\" command that allows the BLV user to go to a particular step in the tour.",
+      "line": 150
+    },
+    {
+      "file": "package.json",
+      "description": "We modified some of the key bindings to make it easier for the BLV user to quickly navigate the code and remove conflicts between those used in CodeTour with the editor navigation keys. ",
+      "line": 620
+    },
+    {
+      "file": "src/store/actions.ts",
+      "description": "This function handles the \"Go To Tour Step\" command from the user. Once the user inputs a number between 1 and the lenght of the step, this functions opens that step and make it the active step. It also announces the active tour to make sure the BLV user knows which step was opened. ",
+      "line": 173
+    },
+    {
+      "file": "src/player/index.ts",
+      "description": "This is where we add the extra context that helps a BLV user know where they are in the file. We make sure to only add the context when the user is not in edit mode (i.e the user is in preview mode), because we don't want the extra context to be saved as part of the tour. ",
```

Comment:
```suggestion
      "description": "This is where we add the extra context that helps a BLV user know where the tour step is within the file. We make sure to only add the context when the user is not in edit mode (i.e the user is in preview mode), because we don't want the extra context to be saved as part of the tour. ",
```